### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by Jessie93** [Jessie93/Weather-Pourcast#76]
Acceptance Criteria

Accessibility: The cards should be accessible to users with disabilities and meet accessibility guidelines, such as proper use of alt text and semantic HTML.
